src/gpodder/gtkui/interface/common.py

```diff
@@ -177,50 +177,63 @@ def show_login_dialog(self, title, message, username=None, password=None,
         if register_callback is not None:
             dialog.add_button(register_text, gtk.RESPONSE_HELP)
 
+        url_entry = gtk.Entry()
+        url_entry.set_tooltip_text(_('hostname or root URL (e.g. https://my.gpo.me)'))
```

Comment:
Would it be better to use https://gpodder.net/ as the example? https://my.gpo.me/ does not load for me.